python_modules/libraries/dagster-aws/dagster_aws/ecs/launcher.py

```diff
@@ -172,7 +173,40 @@ def _task_definition(self, metadata, image):
             task_definition = self.ecs.describe_task_definition(taskDefinition=self.task_definition)
             return task_definition["taskDefinition"]
 
-        return default_ecs_task_definition(self.ecs, metadata, image, self.container_name)
+        return default_ecs_task_definition(
+            self.ecs,
+            metadata,
+            image,
+            self.container_name,
+            secrets=self._dagster_secrets(),
```

Comment:
this is a bit different from how we handle secrets in e.g. K8s - where we have you specify the list of secret names individually.

Using a tag seems reasonable (less config!) but I think we may want to let the user specify the list of tags to use vs. assuming that it's "dagster" and only "dagster"?

Reply:
Made a few updates here based on these suggestions:

- you can now configure the tag name
- you can also pass specific secrets in when configuring the launcher.